### ISSUE TITLE
Table is not refreshed after experiment is removed

### ISSUE THREAD

**Issue by shcheklein** [iterative/vscode-dvc#3103]
On Windows, conda, VS Code 1.74.3, extension 0.5.36:

https://user-images.githubusercontent.com/3659196/212576087-b247156c-8914-4b69-8112-4e6177a402cf.mov

```
[version: 0.5.36, 2023-01-16T00:32:31.286Z, pid: 4972] > C:\Users\ivan\miniconda3\envs\hackathon\python.exe -m dvc exp remove picky-wads - COMPLETED (1133ms)
```

```
DVC version: 2.41.1 (pip)
---------------------------------
Platform: Python 3.10.8 on Windows-10-10.0.22621-SP0
Subprojects:
        dvc_data = 0.29.0
        dvc_objects = 0.14.1
        dvc_render = 0.0.17
        dvc_task = 0.1.9
        dvclive = 1.3.2
        scmrepo = 0.1.5
Supports:
        http (aiohttp = 3.8.3, aiohttp-retry = 2.8.3),
        https (aiohttp = 3.8.3, aiohttp-retry = 2.8.3)
Cache types: hardlink
Cache directory: NTFS on C:\
Caches: local
Remotes: https
Workspace directory: NTFS on C:\
Repo: dvc, git
```

**Comment by Walterosc** [comment]
Also you get the same behavior if you try to remove the experiment by DVC extension left panel (Experiments section).

![image](https://user-images.githubusercontent.com/8837665/212636886-2927bb06-b8eb-4402-abc4-0bf8f1dd9136.png)

**Comment by mattseddon** [comment]
@Walterosc the fix for this will be available in `0.6.1`. Should be available shortly.

**Comment by Walterosc** [comment]
Thanks to all of you @mattseddon.
